week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -15,7 +26,14 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+function validate(num) {
+  let nums = num;
+  if (nums % 2 == 0 && nums <= 100) {
+    return true;
+  } else {
+    return false;
```

Comment:
Nice! One thing you can think about to improve the answer further would be whether all requirements for the `num` variable are expressed well in this `if` statement. Could you think of a way to make the `is a number` be part of the condition?

Reply:
I can use the Number.isNan(num)